### ISSUE TITLE
the train loss is nan

### ISSUE THREAD

**Issue by MrYANG23** [wangziren1/pytorch_pose_proposal_network#4]
hi! thanks for your work ,when  i train this repo i get nan loss no matter use the resnet18 or using the resnet50. can you give me some advices,thank you !

**Comment by PuAnysh** [comment]
Me Too.this problem will occur while i use loss_coor in final loss.

**Comment by wangziren1** [comment]
Maybe, dropping learning rate by ten times or five times helps.
